atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -38,7 +38,7 @@ def get_config(workflow):
 
 
 def get_koji(workflow):
-    return get_config(workflow)['koji']
+    return get_config(workflow).conf['koji']
 
 
 def get_koji_session(workflow):
```

Comment:
What if these `get_*` methods took a fallback value? It might make usage from plugins a little bit easier.

```
def get_koji(workflow, fallback=None):
  try:
    return get_config(workflow).conf['koji']
  except KeyError:
    if fallback:
      return fallback
    raise

def get_koji_session(workflow, fallback):
  config = get_koji(workflow, fallback)
  ...
```

Then the plugins could create a dict with the same format defined in schema:
```
fallback_koji_config = {
  'hub_url': ...
  'auth': {
    'proxyuser': ...
    'ssl_certs_dir': ...
   ...
  }

koji_session = get_koij_session(workflow, fallback_koji_config)
```

It would make most of the repeated pattern of `check_variable` not needed.

Reply:
yes sounds like a good idea